addons/arcadian/base/config_useractions.hpp

```diff
@@ -0,0 +1,17 @@
+class UserActions {
+    class OpenRearWindow {
+        displayName = "Open Rear Window";
+        displayNameDefault = "";
+        position = "";
+        radius = 3.7;
+        onlyForPlayer = 0;
+        priority = 10;
+        condition = "this doorPhase 'reardoor_source' < 0.5 && this doorPhase 'reardoor_2_source' < 0.5 && ((this getCargoIndex player) isEqualTo 0)";
```

Comment:
```suggestion
        condition = "this doorPhase 'reardoor_source' < 0.5 && {this doorPhase 'reardoor_2_source' < 0.5} && {(this getCargoIndex player) isEqualTo 0}";
```
`player` here will not support remote control I think, you should test.

Reply:
It doesn't, same reason sirens and other weird functionality on vehicles don't work when units are RC'd.

Reply:
Well it should. See https://github.com/CBATeam/CBA_A3/blob/master/addons/common/fnc_currentUnit.sqf. Since we don't rely on CBA, you can just use that line of code.